src/main/java/duke/exception/BadDateArgumentException.java

```diff
@@ -0,0 +1,9 @@
+package duke.exception;
+
+public class BadDateArgumentException extends Exception{
+
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1L;
```

Comment:
Perhaps constants should be in screaming snake case? I noticed the same issue at other parts of your code